contrib/perf/get-pr-info.py

```
@@ -0,0 +1,137 @@
+    """
+    Get PR numbers from workflow run via GitHub API.
+
+    Uses the workflow_run.pull_requests field from the API.
+    """
+    url = f"https://api.github.com/repos/{owner}/{repo}/actions/runs/{run_id}"
+    headers = {
+        'Authorization': f'token {token}',
+        'Accept': 'application/vnd.github.v3+json'
+    }
+
+    response = requests.get(url, headers=headers)
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)